src/libraries/System.Private.CoreLib/src/System/String.Comparison.cs

```diff
@@ -1066,5 +1168,20 @@ private static CompareOptions GetCompareOptionsFromOrdinalStringComparison(Strin
             int ct = (int)comparisonType;
             return (CompareOptions)((ct & -ct) << 28); // neg and shl
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static bool VectorContainsNonAsciiChar(Vector128<ushort> utf16Vector)
```

Comment:
such helpers already exists in Ascii.Utility.cs (VectorContainsNonAsciiChar)

Reply:
This method is private so might not be accessed here.

Reply:
This is fine to make that method internal if needed.